deployer/templates/fluentd.yaml

```diff
@@ -63,6 +63,7 @@ objects:
             resources:
               limits:
                 cpu: 100m
+                memory: 512Mi
```

Comment:
The fluentd pods don't use a lot of memory when the flow of logs is not interrupted.  But if the output to Elasticsearch backs up, then the queue can grow and memory will be used.

In hind-sight, the testing of this PR should have been to verify that the buffer queue used for each output filter combined does not exceed the 512 MB limit.